starter-code/src/functions-and-arrays.js

```diff
@@ -60,6 +157,16 @@ var wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist (arrayWords, searchWord){
+  for (var i = 0; i < arrayWords.length; i++){
+    if (searchWord == arrayWords[i]){
+       return true;
+    }
+  }
+  return false;
+}
```

Comment:
Correcto, además habéis escogido for, que es lo suyo, en vez de map( ) o forEach( ), muy bien!
Como nota al margen, podéis usar el método includes( ) para resolver este ejercicio. Es muy útil